### PR TITLE
Docgenerator: Example style change for MusicXML

### PR DESCRIPTION
**Pull request by mdgood**

### CODE REVIEW

File: docgenerator/spec/templates/example_detail.html [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <h1>{{ example.name }}</h1>
 
 {% if example.image_url %}
-<p width="100%"><img src="{% relative_url_string example.image_url %}" width="100%"></p>
+<p><img src="{% relative_url_string example.image_url %}"/></p>
```
Comment (adrianholovaty) [inline docgenerator/spec/templates/example_detail.html:22]: We're not using XHTML for this site, so the slash at the end of  isn't necessary (to be consistent with the rest of the templates).

**Comment by adrianholovaty** [comment]
@mdgood Thanks — all looks good, except would you mind removing those slashes (per my review comment)?

**Comment by mdgood** [comment]
Done. Old XML habits die hard. I've got the slash at the end of all the hard-coded `` elements within the description field.

**Comment by adrianholovaty** [comment]
Fixed this in ce713c49a. Thanks!

**Pull request closed without merging**